Modules/pwdmodule.c

```diff
@@ -183,6 +187,22 @@ pwd_getpwuid(PyObject *module, PyObject *uidobj)
         if (nomem == 1) {
             return PyErr_NoMemory();
         }
+
+// iPhone has a "user" with UID 501, username "mobile"; but the simulator
+// doesn't reflect this. Generate a simulated response.
+#if TARGET_OS_SIMULATOR
+        if (uid == 501) {
+            struct passwd mp;
+            mp.pw_name = "mobile";
+            mp.pw_passwd = "/smx7MYTQIi2M";
```

Comment:
I see it's easy enough to Google, but there should still be a comment explaining what this value is, and why it was chosen.